plugins/events/create.js

```diff
@@ -244,6 +246,23 @@ module.exports = () => ({
             if (event.builds === null) {
                 return boom.notFound('No jobs to start.');
             }
+
+            const virtualJobBuilds = event.builds.filter(b => b.status === 'CREATED');
```

Comment:
is that enough to assume that all builds with `status === 'CREATED'` is a virtual job then?

Reply:
This could be plural, when we have an event started from `~pr` or `~commit`, and there are more than one virtual job listening to these triggers.

Reply:
yes, builds created immediately after the event is created would be in `CREATED` status only if the job is virtual.

Reply:
What about builds that are waiting for other joined parent downstream builds to complete? Aren't those builds normally created and then park with "CREATED" status?

Reply:
There are the builds at the beginning of the workflow for an event. I believe, these wouldn't be waiting for any upstream.